### DISCUSSION TITLE
eslint: Parsing error

### DISCUSSION THREAD

**Discussion by davidarny** [zed-industries/zed#6648] [Feature Requests]
Opening any file in my project results in an error on a first line

<img width="815" alt="image" src="https://github.com/zed-industries/community/assets/17799810/4a8f5a15-e1e1-411c-8e2b-7719f7984eb3">

As far as I understand Zed looks for a `tsconfig` in a directory where file is located, however there is a root `tsconfig` file

<img width="679" alt="image" src="https://github.com/zed-industries/community/assets/17799810/2f22789b-8e64-43cb-8fb9-72d9271918b5">

And everything works well in other IDE's

**Comment by HectorManriquez** [comment]
I'm not sure if you ever got it fixed but this solved it for me. Not related to zed editor but to eslint configuration itself.

https://stackoverflow.com/questions/64933543/parsing-error-cannot-read-file-tsconfig-json-eslint

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
